model/city.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import (
+	"encoding/json"
+	"github.com/hazelcast/hazelcast-go-client/serialization"
+)
+
+type City struct {
+	Country    string `json:"country"`
+	City       string `json:"city"`
+	Population int64  `json:"population"`
+}
+
+func CityAsJson(country string, city string, population int64) serialization.JSON {
```

Comment:
Go convention is using all upper case for initialisms. In this case it would be: `CityAsJSON`.